service/src/test/java/bio/terra/externalcreds/dataAccess/SshKeyPairInternalDaoTest.java

```diff
@@ -63,6 +74,32 @@ void upsertSshKey() throws NoSuchAlgorithmException, IOException {
 
       verifySshKeyPair(sshKey, storedSshKey);
     }
+
+    @Test
+    void encryptKeyFails() {
```

Comment:
once you handle IOException internal to the encrypt and decrypt functions then the failure tests should be removed because they don't really test anything